src/main/scala/gitbucket/core/service/RepositoryService.scala

```diff
@@ -434,7 +434,7 @@ trait RepositoryService { self: AccountService =>
     val preferred = PluginRegistry().renderableExtensions.map(extension => s"${fileBaseName}.${extension}")
 
     def choiceTemplate(files: List[FileInfo]): Option[FileInfo] =
-      files.find(f => preferred.contains(f.name)).orElse(files.find(f => f.name.startsWith(fileBaseName)))
+      files.find(f => preferred.contains(f.name)).orElse(files.find(f => fileBaseName == FileUtil.trimExtension(f.name)))
```

Comment:
Although markup renderer can be added by plugin, the content of issues and pull request are rendered as markdown certainly. So it doesn't need to look `PluginRegistry().renderableExtensions` at here. I think that only the following filenames are sufficient for the target.

- `ISSUE_TEMPALTE`
- `ISSUE_TEMPALTE.md`
- `ISSUE_TEMPALTE.markdown`